### PR TITLE
feat: convert dex_prices to dunesql

### PR DESCRIPTION
**Pull request by m-r-g-t**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

File: models/dex/dex_prices.sql [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{ config(
-    alias = 'prices',
+    alias = alias('prices'),
```
Comment (jeff-dude) [inline models/dex/dex_prices.sql:2]: at top of this config block, add:
`tags=['dunesql'],`

Comment (m-r-g-t) [reply models/dex/dex_prices.sql:3]: added on [39ed84f](https://github.com/duneanalytics/spellbook/pull/3684/commits/39ed84f0f28fcb768756a617a4aad760f2f5b739)

File: models/dex/dex_prices.sql [hunk 2]
```diff
@@ -8,57 +9,56 @@
...
 FROM 
 (
     SELECT 
-        date_trunc('hour', block_time) as hour, 
+        date_trunc('hour', block_time) AS hour, 
         contract_address,
         blockchain,
-        (PERCENTILE_CONT(0.5) WITHIN GROUP (ORDER BY price)) AS median_price,
-        COUNT(price) as sample_size 
+        approx_percentile(price, 0.5) AS median_price,
```
Comment (jkylling) [inline models/dex/dex_prices.sql:60]: Why do we change to `approx_percentile` here?

**Comment by jeff-dude** [comment]
we've also agreed that this has to wait until `dex.trades` is fully migrated. let us know if you think otherwise @jkylling

**Comment by jeff-dude** [comment]
closing since `dex.prices` has been migrated

**Pull request closed without merging**